src/clients/python/tests/test_basic.py

```diff
@@ -0,0 +1,1348 @@
+import os
```

Comment:
argh, this is unfortunate a bit, as node tests are not written in the correct style. Specifically, hard-coding account ids like we do here is not great, because this makes tests order-dependent. 

Both sets of test should have used `tbid` function to generate unique ids. But I guess we will then refactor both sets of tests later?

Reply:
Or, I guess, we should be _generating_ the tests?